frontend/app/src/components/import/GroupedImport.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <card>
+    <div class="pa-2">
+      <v-select
+        v-model="selectedSource"
+        :label="$t('import_data.select_source.title')"
+        outlined
+        :items="sources"
+        item-value="identifier"
+        item-text="name"
+        :hide-details="true"
+      >
+        <template #item="{ item }">
+          <v-row v-if="item">
+            <v-col class="d-flex align-center justify-space-between col-12">
+              <div
+                :class="`${$style.image} ${
+                  dark ? $style['image--inverted'] : ''
```

Comment:
wouldn't the [object syntax](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax) make more sense here?